src/System.Memory/tests/System.Memory.Tests.csproj

```diff
@@ -0,0 +1,43 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(NetFrameworkMinimum)</TargetFramework>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <!-- Don't run .NET Framework only tests on non Windows. -->
+    <SkipTests Condition="'$(OS)' != 'Windows_NT'">true</SkipTests>
```

Comment:
Now that we have more than one test project, it would be good to move this condition out into the root Directory.Build.targets file and condition it on `"'$(TargetFrameworkIdentifier)' == '.NETFramework' and '$(OS)' != 'Windows_NT'"`. Remove the line from the other test project then.

Reply:
Sure. I also noticed that we will need AllowUnsafe to apply to all test projects because we are now importing PlatformDetection in all tests projects, and that type requires it.